activerecord/lib/active_record/relation/delegation.rb

```diff
@@ -89,6 +89,9 @@ def method_missing(method, *args, &block)
             self.class.delegate_to_scoped_klass(method)
             scoping { @klass.public_send(method, *args, &block) }
           elsif arel.respond_to?(method)
+            ActiveSupport::Deprecation.warn \
+              "Delegating #{method} to arel is deprecated and will be removed in Rails 5.3. " \
+              "Use relation.arel.#{method} instead."
```

Comment:
Right. Better to say that is going to be removed with no replacement. Also the next version of Rails after 5.2 is 6.0